src/Security/Role.php

```diff
@@ -21,10 +35,14 @@ class Role extends Document
      * @param string $id Unique role identifier
      * @param array $content Role content
      * @param array $meta Role metadata
+     * @return Role
      */
     public function __construct(Security $kuzzleSecurity, $id = '', array $content = [], array $meta = [])
     {
-        parent::__construct($kuzzleSecurity, $id, $content, $meta);
+        $this->security = $kuzzleSecurity;
```

Comment:
In https://github.com/kuzzleio/sdk-javascript/blob/6.x/src/controllers/security/role.js, it's a Kuzzle object instead of Security. Should I do the same here ?